containersQa/testlib.bash

```diff
@@ -630,6 +630,39 @@ function s2iBinaryCopy() {
   rm -rf $d
 }
 
+function s2iHsPerfDataBuild() {
+  prepareS2I
+  setUser
+  if [ "x$s2iBin" == "xundef" -o "x$s2iBin" == "x" ] ; then
+    echo $SKIPPED
+    return
+  fi
+  local d=`mktemp -d`
+  pushd $d
+    DF=s2iDockerFile
+
+    $s2iBin build --pull-policy never --as-dockerfile $DF --context-dir=$CONTEXTDIR -r=${rev} \
+                   --assemble-user 185 --loglevel 1 --image-scripts-url image:///usr/local/s2i "$APP_SRC" "$BASEIMG" "$OUTIMG"
```

Comment:
I must confess I do not parse.   This is building random application? What application?

Reply:
Basically, this is similar to the previous s2i builds. But this is building one of the sample applications from the repo I pointed out below. The goal of this test is to verify that the hsperfdata_$USER folder gets removed from the /tmp folder once the building is complete. This has been working on the ubi8 side and was missed on the ubi9 side when the user changed from jboss to default.

Reply:
hmhmhhm. Can not the other tests be used? See this only as brainstroming - but about 5 s2i are running in previous tests. hdper data should be missing on all of them, right?

Reply:
technically it should. the other tests would need to be changed so checking for this could be possible. right now all those images are nameless on the host under test.